portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,241 @@
+.align-left {
+    text-align: left;
+    width: 90%;
+}
+
+.big-margin-bottom {
+    margin-bottom: 200px !important;
+}
+
+.bigger-margin-bottom {
+    margin-bottom: 280px !important;
+}
+
+.blog-images {
+    width: 55%;
+    flex-shrink: 1;
```

Comment:
good practice is to order the properties in alphabetical order. Please do this for all selectors

Reply:
Ordered css properties alphabetically.